pkg/pillar/pubsub/socketdriver/driver.go

```diff
@@ -87,8 +81,8 @@ func (s *SocketDriver) Publisher(global bool, name, topic string, persistent boo
 	// the dirName depends on if we are persistent, and if it is the global config
 	switch {
 	case persistent && publishToDir:
-		// Special case for /persist/config/
-		dirName = fmt.Sprintf("%s/%s/%s", s.RootDir, persistConfigDir, name)
+		// No longer supported
+		return nil, errors.New("Persistent not supported for empty agentname")
```

Comment:
I'm trying to understand what are the "global" subscriptions.
I've tried to find any "anonymous" (without agentname) Publisher for example but I don't see any...

Reply:
I see two on a running system by looking in /run/global/:
DevicePortConfig  LedBlinkCounter

For LedBlinkCounter there is only a subscriber I think - some old legacy code to push changes to that config.

Reply:
Some of it is documented [here](https://github.com/lf-edge/eve/blob/master/docs/IPC.md) @OhmSpectator

Reply:
@deitch, oh, thanks! I wish I had found this doc much earlier.